app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -14,7 +14,7 @@ def start
     else
       load_nominate_induction_tutor_form
       @nominate_induction_tutor_form.token = token
-      session[:nominate_induction_tutor_form] = @nominate_induction_tutor_form
+      session[:nominate_induction_tutor_form] = @nominate_induction_tutor_form.as_json
```

Comment:
This is interesting - AR storage actually deserialized the entire from, so `session[:nominate...]` was an instance of the form object in the subsequent requests. I've maintained the original logic, but potentially we could just use this.

Reply:
I prefer `as_json`, I think storing the form was a bit of an oversight.

Reply:
Definitely. Cookie store was implicitly casting the form onto json. I think I'd prefer `attributes` (from ActiveModel::Attributes) here as `as_json` is not that well defined. But it is close enough! :)